dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNVscConverterInterfaceIIDM.h

```diff
@@ -0,0 +1,149 @@
+//
+// Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+// simulation tool for power systems.
+//
+
+//======================================================================
+/**
+ * @file  DYNVscConverterInterfaceIIDM.h
+ *
+ * @brief Vsc converter interface : header file for IIDM implementation
+ *
+ */
+//======================================================================
+#ifndef MODELER_DATAINTERFACE_POWSYBLIIDM_DYNVSCCONVERTERINTERFACEIIDM_H_
+#define MODELER_DATAINTERFACE_POWSYBLIIDM_DYNVSCCONVERTERINTERFACEIIDM_H_
+
+#include "DYNVscConverterInterface.h"
+
+#include "DYNInjectorInterfaceIIDM.h"
+
+#include <powsybl/iidm/VscConverterStation.hpp>
+
+#include <boost/shared_ptr.hpp>
+
+namespace DYN {
+
+/**
+ * class VscConverterInterfaceIIDM
+ */
+class VscConverterInterfaceIIDM : public VscConverterInterface, public InjectorInterfaceIIDM {
+ public:
+  /**
+   * @brief Destructor
+   */
+  ~VscConverterInterfaceIIDM() {}
+
+  /**
+   * @brief Constructor
+   * @param vsc: vsc converter iidm instance
+   */
+  explicit VscConverterInterfaceIIDM(powsybl::iidm::VscConverterStation& vsc);
+
+  /**
+   * @copydoc ComponentInterface::importStaticParameters()
+   */
+  void importStaticParameters();
+
+  /**
+   * @copydoc ComponentInterface::exportStateVariablesUnitComponent()
+   */
+  void
```

Comment:
same as above. The coding style makes a difference between declaration and definition. We should keep the difference instead of changing coding style between cpp and hpp files. If I change it, next time I launch clang-format it will turn again to that version.